application/src/test/java/org/opentripplanner/apis/gtfs/GraphQLFormattingTest.java

```diff
@@ -7,12 +7,13 @@
 import graphql.schema.idl.SchemaPrinter;
 import java.io.File;
 import org.junit.jupiter.api.Test;
+import org.opentripplanner.test.support.ResourceLoader;
 
 public class GraphQLFormattingTest {
 
-  public static final File SCHEMA_FILE = new File(
-    "src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls"
-  );
+  public static final File SCHEMA_FILE = ResourceLoader.of(
+    GraphQLFormattingTest.class
+  ).mainResourceFile("schema.graphqls");
```

Comment:
If I remember correctly, there was some reason why I didn't use the resource loader here (maybe because this depends on the built resources). Were these types of issues somehow fixed with this mainResourceFile method? Although, it could be that I just copied the approach from the transmodel API without paying any attention.

Reply:
I think the method didn't exist back then.